presto-spi/src/main/java/com/facebook/presto/spi/type/UnscaledDecimal128Arithmetic.java

```diff
@@ -162,6 +181,13 @@ public static long unscaledDecimalToUnscaledLong(Slice decimal)
         return negative ? -lo : lo;
     }
 
+    public static long unscaledDecimalToUnscaledLongUnsafe(Slice decimal)
+    {
+        long lo = getLong(decimal, 0);
```

Comment:
let's make it separate patch to make it consistent after our PR are merged into dev branch. Let's us low/high from now